src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -403,6 +404,24 @@ public SnapshotAccessRequestResponse approveRequest(UUID id) {
     return snapshotRequestDao.getById(id).toApiResponse();
   }
 
+  public SnapshotAccessRequestMembersResponse getGroupMembers(UUID id) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    return new SnapshotAccessRequestMembersResponse()
+        .members(iamService.getGroupPolicyEmails(model.samGroupName(), "member"));
+  }
+
+  public SnapshotAccessRequestMembersResponse addGroupMember(UUID id, String memberEmail) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    return new SnapshotAccessRequestMembersResponse()
+        .members(iamService.addEmailToGroup(model.samGroupName(), "member", memberEmail));
+  }
+
+  public SnapshotAccessRequestMembersResponse deleteGroupMember(UUID id, String memberEmail) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    return new SnapshotAccessRequestMembersResponse()
+        .members(iamService.removeEmailFromGroup(model.samGroupName(), "member", memberEmail));
```

Comment:
For all of these, do we only want people with the member role? Is there a way to get people of any role?

Reply:
We should only ever add users as members to the sam group. Only the TDR service account will be an admin.